configs/deploy/single-stage_ncnn_static-800x1344.py

```diff
@@ -0,0 +1,4 @@
+_base_ = '../_base_/base_static.py'
+
+codebase_config = dict(model_type='ncnn_end2end')
+onnx_config = dict(output_names=['detection_output'], input_shape=[1344, 800])
```

Comment:
No model in mmyolo use this resolution. Is this config necessary?

Reply:
> No model in mmyolo use this resolution. Is this config necessary?

This is from the original config file in mmdeploy/configs/mmdet/detection/single-stage_ncnn_static-800x1344.py.